angular/projects/admin/src/app/admin/document/document-edit/document-edit.component.html

```diff
@@ -50,13 +50,10 @@
                               formControlName="{{field.key}}"></textarea>
                   </mat-form-field>
 
-                  <mat-form-field *ngSwitchCase="'textbox-rich'"
-                                  class="full-width">
-                    <tanam-textbox-rich matInput
-                                        placeholder="{{field.title}}"
-                                        [editorConfig]="richTextEditorConfig"
-                                        formControlName="{{field.key}}"></tanam-textbox-rich>
-                  </mat-form-field>
```

Comment:
How does it affect the page form's validation and change detection if we remove the `<mat-form-field>` wrapper?

Reply:
![Peek 2019-08-16 17-49](https://user-images.githubusercontent.com/18025983/63159581-784ff980-c04e-11e9-8054-a708c1ac7d5c.gif)

Form validation works fine without `<mat-form-field>` wrapper